public/stylesheets/site/2.0/07-interactions.css

```diff
@@ -179,6 +179,7 @@ We only use error messages for LiveValidation. Style spoofs the system error mes
 .LV_validation_message {
   font-weight: 900;
   position: absolute;
+  margin-right: 15em;
```

Comment:
I think we also need to adjust the right margin on `.LV_invalid` below, or else this will get overridden when displaying an error message.

Reply:
Haven't forgotten about this, just haven't had a chance to do this + test visually. Will do though.

Reply:
No worries! Thanks for letting us know.

Reply:
I've updated the css so all the changes are on `.LV_invalid` only.